gatsby-ssr.js

```diff
@@ -1,14 +0,0 @@
-import React from 'react'
```

Comment:
Pretty sure using both `allPageHeaders` in `gatsby-config` options for `gatsby-plugin-gatsby-cloud` and the `setHeadComponents` in `gatsby-ssr` are superfluous.

Seems like the `gatsby-plugin-gatsby-cloud` might be preferable from a performance perspective, but I don't have any data to support that.